ts/components/FiscalCodeComponent.tsx

```diff
@@ -449,7 +452,29 @@ export default class FiscalCodeComponent extends React.Component<Props> {
 
   public render(): React.ReactNode {
     return (
-      <View>
+      <View
+        accessible={true}
+        accessibilityLabel={
```

Comment:
lgtm
To improve _render_ readability can could we refactor as follows?

```suggestion
        accessibilityLabel={this.accessibilityText.accessibilityLabel}
        accessibilityHint={this.accessibilityText.accessibilityHint}
```

```typescript
get accessibilityText(): Record<
    "accessibilityLabel" | "accessibilityHint",
    string
  > {
    if (this.props.type === "Preview") {
      return {
        accessibilityLabel: I18n.t(
          "profile.fiscalCode.accessibility.preview.label"
        ),
        accessibilityHint: I18n.t(
          "profile.fiscalCode.accessibility.preview.hint"
        )
      };
    }
    const isLandScape = this.props.type === "Landscape";
    if (this.props.getBackSide) {
      return {
        accessibilityLabel: I18n.t(
          "profile.fiscalCode.accessibility.rear.label"
        ),
        accessibilityHint: isLandScape
          ? ""
          : I18n.t("profile.fiscalCode.accessibility.rear.hint")
      };
    }
    // goBackSide === false
    return {
      accessibilityLabel: I18n.t(
        "profile.fiscalCode.accessibility.front.label",
        {
          code: this.props.profile.fiscal_code,
          name: this.props.profile.name,
          family_name: this.props.profile.family_name
        }
      ),
      accessibilityHint: isLandScape
        ? ""
        : I18n.t("profile.fiscalCode.accessibility.front.hint")
    };
  }
```

Reply:
added in ca541de